tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -405,5 +405,33 @@ public class LinkModeFullClass {
 			}
 		}
 
+		[Test]
+		public void AndroidAotModeHybridAbortsOnArmeabiV7a ()
+		{
+			AssertHasDevices ();
+
+			var proj = new XamarinAndroidApplicationProject () {
+				IsRelease = true,
+				AotAssemblies = true,
+			};
+			proj.SetProperty ("AndroidAotMode", "Hybrid");
+			proj.SetAndroidSupportedAbis ("armeabi-v7a");
```

Comment:
Ha. I just realized this probably won't work on the test bots because I think the tests only run on x86_64 emulator. I'll take another look at it tomorrow.